tensorflow/core/common_runtime/process_util.cc

```diff
@@ -101,7 +101,7 @@ int32 NumIntraOpThreadsFromEnvironment() {
   const char* val = std::getenv("TF_NUM_INTRAOP_THREADS");
   return (val && strings::safe_strto32(val, &num)) ? num : 0;
 }
-
+#ifndef ENABLE_MKLDNN_THREADPOOL
```

Comment:
Nit: Since this and `INTEL_MKL` start and end together, would you mind merging them so it's easier to understand?
```suggestion
#if defined(INTEL_MKL) && !defined(ENABLE_MKLDNN_THREADPOOL)
...
#endif  // defined(INTEL_MKL) && !defined(ENABLE_MKLDNN_THREADPOOL)
```

Same for a few other places in this file and `threadpool_device.cc`.